pom.xml

```diff
@@ -169,6 +174,7 @@
                                 <compileSourceRoots>
                                     <compileSourceRoot>${java21.sourceDirectory}</compileSourceRoot>
                                 </compileSourceRoots>
+                                <outputDirectory>${project.build.outputDirectory}/META-INF/versions/21</outputDirectory>
```

Comment:
- #121

Reply:
Fixed, thanks!
I didn't know about this.